examples/application_sql_with_custom_transformer.py

```diff
@@ -116,11 +119,23 @@ def get_attributes(cls, obj: Dict[str, Any]) -> Dict[str, Any]:
         }
 
 
-class CustomTransformer(AtlasTransformer):
+class CustomTransformer(QueryBasedTransformer):
     def __init__(self, connector_name: str, tenant_id: str, **kwargs: Any):
         super().__init__(connector_name, tenant_id, **kwargs)
 
-        self.entity_class_definitions["DATABASE"] = PostgresDatabase
+        self.entity_class_definitions = get_yaml_query_template_path_mappings(
+            custom_templates_path=os.path.join(
+                os.path.dirname(__file__), "sql_query_templates"
+            ),
+            assets=[
+                "TABLE",
+                "COLUMN",
+                "DATABASE",
+                "SCHEMA",
+                "EXTRAS-PROCEDURE",
+                "FUNCTION",
+            ],
```

Comment:
nitpick: can you add a comment here explaining the type (i.e Database) will be overrided?

Reply:
Done